reference/volumes.html.md.erb

```diff
@@ -109,7 +107,7 @@ Created at: 04 Mar 23 03:32 UTC
 
 ### Mount a Volume
```

Comment:
I'm not sure about including how to mount or access a volume in the volumes reference doc. `fly.toml` is app-platform-as-a-service (i.e. `fly deploy`) -specific. You can mount volumes to non-platform Machines (in apps that have no `fly.toml`) too. I think it's fine to point to other docs from here though.

Reply:
OK, I'll leave Access a Volume with some links and remove the Mount a Volume section.